SNIPS/snip-6.md

```diff
@@ -85,6 +123,12 @@ Wallets are advised not to sign any data unless they know that it includes the a
 
 ## Backwards Compatibility
 
+As mentioned before the interface ID for an account is always `0x2ceccef7f994940b3962a6c67e0ba4fcd37df7d131417c604f91e03caecc1cd`** which matches the original account interface. 
+
+Note that the the current interface in and the legacy one are compatible in the way they can be called.
+
+The latest `__execute__` execute methods returns no data. But older versions used to return the result of the calls. 3rd parties shouldn't rely on the data being returned
```

Comment:
```suggestion
The latest `__execute__` methods return no data. But older versions return the result of the calls. Third parties shouldn't rely on the data being returned.
```

Reply:
i didn't apply as is, but i fixed the typos